tests/amm_test.rs

```diff
@@ -645,14 +645,14 @@ mod e2e_amm {
             .await;
 
         info!("➡️  Sending recursive proof for hydentity, amm, hyllar and hyllar2");
-        ctx.send_recursive_proof(
+        ctx.send_proof(
             "risc0-recursion".into(),
             ProofData::Bytes(recursive_proof),
             vec![
-                (blob_tx_hash.clone(), "hydentity".into()),
-                (blob_tx_hash.clone(), AMM_CONTRACT_NAME.into()),
-                (blob_tx_hash.clone(), "hyllar".into()),
-                (blob_tx_hash.clone(), "hyllar2".into()),
+                blob_tx_hash.clone(), // "hydentity".into()),
+                blob_tx_hash.clone(), // AMM_CONTRACT_NAME.into()),
+                blob_tx_hash.clone(), // "hyllar".into()),
+                blob_tx_hash.clone(), // "hyllar2".into()),
```

Comment:
I find this usage weird, could be improved later but fine

Reply:
Not sure if you were referring to the comments or just the repeated Tx hash. Until we parse that from the proof output itself we have to list it, it just happens to be the same blob TX everywhere here

Reply:
I was referring to the repeated tx hash

Reply:
Then yeah I think this just highlights a limitation of our current code